src/components/Round/Round.tsx

```diff
@@ -45,12 +56,18 @@ export const Round: FC<Props> = ({ lang, choices, onAnswer }) => {
   useEffect(() => {
     if (!answer) return;
     onAnswer(answer, time);
+    setShowHint(false);
   }, [answer]);
 
   const onTimerChange = (t: number) => {
     setTime(t);
   };
 
+  const handleHint = (s: string) => {
+    let split = s.split('.');
+    return `${split[1].replace(langInfo.title, "this language")}.`;

```

Comment:
Better to use `replaceAll`
```suggestion
    return `${split[1].replace(langInfo.title, "this language")}.`;

```

Reply:
One more things is to add a condition if(split.length > 0) to avoid errors in case no info is available